### ISSUE TITLE
Printables does not contain replacement character

### ISSUE THREAD

**Issue by davetapley** [pyparsing/pyparsing#392]
Here's a awkward one that just cost me a lot of time 😞 

I'm using this pattern to skip over lines I don't care about:
```py
knownLines = # some parser
otherStuff = Suppress(Word(printables))
parser = OneOrMore(knownLines | otherStuff)
```

It wasn't working for some inputs which come from web requests.
Turns out those inputs include one or more �, aka [REPLACEMENT CHARACTER](https://en.wikipedia.org/wiki/Specials_(Unicode_block)),
and since that isn't in `printables` the whole thing broke down.

---

I can work around it with:
```py
otherStuff = Suppress(Word(printables) | Char('�'))
```

But is there any chance you'd consider adding � to `printables`?
https://github.com/pyparsing/pyparsing/blob/f967ba05f24ea74c2c85a87a3423cd3265d4cf10/pyparsing/core.py#L257

**Comment by ptmcg** [comment]
`printables` is just a string, so you could simplify your `otherStuff` expression to:

    otherStuff = Suppress(Word(printables + '�'))

Short answer: I am probably not going to add this Unicode character to the base printables string, since that string has historically limited to characters up to 255.

However, that character _is_ in `pyparsing_unicode.printables`, which also includes many language characters, emojis. wingdings, etc.

Looking a little further now, I might add a Unicode subset that contains most of the language characters but leaves out emojis and wingdings and other symbols. The Basic Multilingual Plane consists of the characters up to ordinal 65535, and this includes your REPLACEMENT CHARACTER character. Using this subset would avoid the startup cost of importing the full pyparsing_unicode printables, which evaluates over a million characters to sort them into alphas, nums, identchars, etc., while (I think!) still including most of the characters for parsing modern language. Needs a little more research.

Thanks for posting this.

**Comment by ptmcg** [comment]
BMP unicode_set released in 3.0.9, including REPLACEMENT CHARACTER
